src/assets/i18n/en.json

```diff
@@ -0,0 +1,226 @@
+{
+  "abnormalities":"Abnormalities",
```

Comment:
http://www.differencebetween.info/difference-between-anomaly-and-abnormality
* Abnormality: usually in context of behavior
* Anomality: usually in context of data or object.

J'ai l'impression que Anomality/anomalities serait la traduction la plus juste.